recipes/tesseract/all/conanfile.py

```diff
@@ -61,7 +61,7 @@ def requirements(self):
             self.requires("libtiff/4.6.0")
         # libarchive is required for 4.x so default value is true
         if self.options.get_safe("with_libarchive", default=True):
-            self.requires("libarchive/3.7.2")
+            self.requires("libarchive/3.7.6")
```

Comment:
```suggestion
            self.requires("libarchive/[>=3.7 <3.8]")
```

This way it can mitigate some conflicts and use 3.7.7 when we publish it

Reply:
Absolutely!